packages/react-components/src/components/timegraph-output-component.tsx

```diff
@@ -317,12 +317,11 @@ export class TimegraphOutputComponent extends AbstractTreeOutputComponent<Timegr
         return true;
     }
 
-    private doHandleSelectionChangedSignal(payload: { [key: string]: string }) {
+    private doHandleItemSelectedSignal(payload: { [key: string]: string; }): void {
         const startTimestamp = payload['startTimestamp'];
         const endTimestamp = payload['endTimestamp'];
         if (startTimestamp !== undefined && endTimestamp !== undefined) {
             const foundElement = this.findElement(payload);
```

Comment:
The vertical sync is not working anymore. The `findElement` method always returns `undefined`. The reason is now the payload has now attribute `load` anymore and it skips over the corresponding if statement (`if (payload && payload.load)`) in `findElement`.